### PR TITLE
Enhancement: Implement bubble sort

### LINKED ISSUE #9: ENH: Implement bubble sort
We currently have a small implementation of a merge sort algorithm only.  We would like to also provide a `bubble_sort` implementation.

For information on bubble sort, please see: https://en.wikipedia.org/wiki/Bubble_sort

### Working on this issue

Implementing bubblesort consists of the following things:
- Find the existing `bubble_sort` stub and replace it with your implementation.
- Add documentation (you can look at `merge_sort` for inspiration).
- Add some tests by expanding `test_bubble_sort` in `test_sorting.py`.

Once you have made your changes, and committed them to a new branch, you can create a PR!

### PR DESCRIPTION
**Pull request by andrea-petrocchi**
Fix #9.

### CODE REVIEW

File: src/listwiz/sorting.py [hunk 1]
```diff
@@ -47,8 +47,27 @@ def merge_sort(l):
 
 
 def bubble_sort(l):
-    # We should provide bubble sort as well!
-    raise NotImplementedError
+    """Bubble sort algorithm.
+
```
Comment (seberg) [inline src/listwiz/sorting.py:51]: A bigger project might want more documentation, but this is great!

**Review by seberg — APPROVED** [review]
Thank you this looks great.  Made sure to have a test and all boxes are ticked :).  In bigger projects/more complex, you would still have to expect some iteration/fixups, unfortunately.

But if I see a PR with this care, I would be very happy! :).

**Comment by seberg** [comment]
@andrea-petrocchi, sorry that I will close this now without merging.  Unfortunately, I looked at the other PR first, which collided.  (And of course you even commented on the issue!)
If this was a bigger project, we should probably made sure that you do get some ackowledgement (e.g. a at least "Co-authored-by") so that your work is not lost.

**Comment by andrea-petrocchi** [comment]
@seberg thank you for your comments. I also saw the other PR being completed before mine. In cases like this is it good practice to close the PR myself or to let the reviewer do that?

**Comment by seberg** [comment]
I think it is great if you just take initiative in such cases.

**Pull request closed without merging**